OpenRA.Mods.Common/Traits/Buildings/Bib.cs

```diff
@@ -74,4 +116,12 @@ public void RemovedFromWorld(Actor self)
 				rs.Remove("bib_{0}".F(i));
 		}
 	}
+
+	public class HideBibPreviewInit : IActorInit<bool>, ISuppressInitExport
```

Comment:
Please use that also for the colour picker in the lobby. It looks ugly with the bib.